testutils/src/main/scala/za/co/absa/enceladus/testutils/e2eSparkRunner/E2ESparkRunner.scala

```diff
@@ -69,8 +67,8 @@ object E2ESparkRunner {
     log.debug(confComparisonRes)
     log.info("Conformance Comparison Passed")
 
-    val humanReadableStdTime = f"${stdTime / 1000 / 60} min ${stdTime  / 1000.0 % 60.0}%.2f s"
-    val humanReadableConfTime = f"${confTime / 1000 / 60} min ${confTime / 1000.0 % 60.0}%.2f s"
+    val humanReadableStdTime = f"${stdTime / 1000 / 60} min ${stdTime / 1000 % 60.0}%.2f s"
```

Comment:
Don't mean to seem too pedantic, but what if a process takes more than an hour? The line will look like `185 min 11 s`, isn't it?

Consider extracting a function that prettifies elapsed time given a millisecond input and use it everywhere.

Reply:
If the elapsed time conversion function becomes very good, I'd likely use it as well for performance measurement/logging

Reply:
I knew why I waited before merging! I can play with it a bit more, let's see what happens 😃